pom.xml

```diff
@@ -64,6 +64,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <jboss.bom.version>6.3.2.GA</jboss.bom.version>
+        <url-pattern>/*</url-pattern>
```

Comment:
Could you put here more descriptive name, please? I'd suggest to use corsUrlPattern